src/EntityFrameworkCoreMock.Moq/EntityFrameworkCoreMock.Moq.csproj

```diff
@@ -3,12 +3,12 @@
   <PropertyGroup>
     <Title>EntityFrameworkCoreMock.Moq</Title>
     <Description>Easy Mock wrapper for mocking EntityFrameworkCore 5 (EFCore5) DbContext and DbSet using Moq</Description>
+    <TargetFrameworks>netstandard2.1;net6.0;net8.0</TargetFrameworks>
```

Comment:
This is defined in https://github.com/huysentruitw/entity-framework-core-mock/blob/develop/src/Directory.Build.props